src/app/[locale]/(pages)/blocco-accesso/operazione-completata/page.tsx

```diff
@@ -61,7 +61,7 @@ const ProfileBlock = (): React.ReactElement => {
       />
       <CopyCodeCard code={addSpacesEvery3Chars(unlockCode)} />
 
-      {isIDPKnown && <IdpListOnApp />}
+      {isIDPKnown === 'true' && <IdpListOnApp />}
```

Comment:
I'll change `isIDPKnown` to `isIdpKnown`. I've seen many time the use of `Idp` so I suggesto to be consistent. What do you think?

Reply:
name changed to isIdpKnown